troposphere/static/js/components/projects/resources/link/details/sections/ExternalLinkActions.react.js

```diff
@@ -16,11 +16,11 @@ define(function (require) {
 
     onDelete: function () {
       var project = this.props.project,
-        link = this.props.link;
+        external_link = this.props.link;
 
       modals.ExternalLinkModals.destroy({
-        link: this.props.link,
-        project: this.props.project
+        project,
```

Comment:
Just FYI, This is something we can do now with ES2015.
`{ name: name }` = `{ name }`